### PR TITLE
Add support for sar:polarizations, code cleanup

### PR DESCRIPTION
**Pull request by lucyhutcheson**
**Related Issue(s):**

- n/a

**Proposed Changes:**

1. Add support for sar:polarization if available for collection

**PR Checklist:**

- [X]  CHANGELOG entry is not required.

### CODE REVIEW

File: src/components/Search/Search.js [hunk 1]
```diff
@@ -277,17 +282,25 @@ const Search = () => {
...
     resultFootprintsRef.current.clearLayers()
   }
 
-  const getQueryVal = () => ({ 'eo:cloud_cover': { gte: 0, lte: _cloudCover } })
+  const getCloudCoverQueryVal = () => ({
+    'eo:cloud_cover': { gte: 0, lte: _cloudCover }
+  })
+
+  const getPolarizationQueryVal = () => ({
+    'sar:polarizations': { eq: ['VV', 'VH'] }
```
Comment (philvarner) [inline src/components/Search/Search.js:302]: now let's hope this works 🤞🏻

Comment (philvarner) [reply src/components/Search/Search.js:302]: unfortunately it doesn't, but `in` does

```suggestion
    'sar:polarizations': { 'in': ['VV', 'VH'] }
```

**Comment by philvarner** [comment]
only one change and I think this will work -- but I haven't gotten a chance to try it yet, so let's wait on that.

File: src/components/Search/Search.js (hunk 1 above)
Comment (lucyhutcheson) [reply src/components/Search/Search.js]: This has been updated.

File: src/components/PopupResult/PopupResult.js [hunk 2]
```diff
@@ -32,7 +32,8 @@ const PopupResult = (props) => {
   )?.href
 
   const cloudCover = props.result?.properties['eo:cloud_cover']
-
+  const polarizations = props.result?.properties['sar:polarizations']
+  console.log(props.result)
```
Comment (philvarner) [inline src/components/PopupResult/PopupResult.js:36]: remove console log

Comment (lucyhutcheson) [reply src/components/PopupResult/PopupResult.js]: It's been removed.

**Review by philvarner — APPROVED** [review]

**Pull request merged**